core/src/main/java/org/apache/calcite/util/format/FormatElementEnum.java

```diff
@@ -54,6 +61,12 @@ public enum FormatElementEnum implements FormatElement {
       sb.append(work.eeeeFormat.format(date));
     }
   },
+  day("eeee", "The full lower case day name") {
```

Comment:
Enum constants should be all uppercase I believe. 
https://docs.oracle.com/javase/tutorial/java/javaOO/enum.html